CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing to Bystro
+
+[WIP]
+
+Welcome to Bystro! We're glad you're interested in contributing to our project. Before you start,
+please take a moment to familiarize yourself with our approach and guidelines.
+
+## Installing Bystro
+Please see our [installation guide](INSTALL.md).
+
+## Contributing a PR
```

Comment:
Do we want to describe the forking model for all contributions since we decided on that as a generalized approach? Or add something like that to the style guide contribution portion?

Reply:
The forking model can be taken for granted for 3rd party contributors.  For 1st party contributions I would prefer to wait for the dust to settle.  We (or I, am least) don't think I've yet found all the snags that can occur when one forks a repo one has commit rights to.